tests/invertmsrc_test.cpp

```diff
@@ -371,130 +363,131 @@ int main(int argc, char **argv)
 
   // stop the timer
   time0 += clock();
   time0 /= CLOCKS_PER_SEC;
 
-  printfQuda("\nDone: %i iter / %g secs = %g Gflops, total time = %g secs\n",
-	 inv_param.iter, inv_param.secs, inv_param.gflops/inv_param.secs, time0);
-
-   for(int i = 0; i < inv_param.num_src ; i++){
-     invertQuda(spinorOutMulti[i], spinorIn[i], &inv_param);
-   }
-
-//  if (true) {
-//    if (inv_param.mass_normalization == QUDA_MASS_NORMALIZATION) {
-//      errorQuda("Mass normalization not supported for multi-shift solver in invert_test");
-//    }
-//
-//    void *spinorTmp = malloc(V*spinorSiteSize*sSize*inv_param.Ls);
-//
-//    printfQuda("Host residuum checks: \n");
-//    for(int i=0; i < inv_param.num_src; i++) {
-//      ax(0, spinorCheck[i], V*spinorSiteSize, inv_param.cpu_prec);
-//
-//      if (dslash_type == QUDA_TWISTED_MASS_DSLASH || dslash_type == QUDA_TWISTED_CLOVER_DSLASH) {
-//	if (inv_param.twist_flavor != QUDA_TWIST_SINGLET)
-//	  errorQuda("Twisted mass solution type not supported");
-//        tm_matpc(spinorTmp, gauge, spinorOutMulti[i], inv_param.kappa, inv_param.mu, inv_param.twist_flavor,
-//                 inv_param.matpc_type, 0, inv_param.cpu_prec, gauge_param);
-//        tm_matpc(spinorCheck[i], gauge, spinorTmp, inv_param.kappa, inv_param.mu, inv_param.twist_flavor,
-//                 inv_param.matpc_type, 1, inv_param.cpu_prec, gauge_param);
-//      } else if (dslash_type == QUDA_WILSON_DSLASH || dslash_type == QUDA_CLOVER_WILSON_DSLASH) {
-//        wil_matpc(spinorTmp, gauge, spinorOutMulti[i], inv_param.kappa, inv_param.matpc_type, 0,
-//                  inv_param.cpu_prec, gauge_param);
-//        wil_matpc(spinorCheck[i], gauge, spinorTmp, inv_param.kappa, inv_param.matpc_type, 1,
-//                  inv_param.cpu_prec, gauge_param);
-//      } else {
-//        printfQuda("Domain wall not supported for multi-shift\n");
-//        exit(-1);
-//      }
-//
-//      axpy(inv_param.offset[i], spinorOutMulti[i], spinorCheck, Vh*spinorSiteSize, inv_param.cpu_prec);
-//      mxpy(spinorIn, spinorCheck, Vh*spinorSiteSize, inv_param.cpu_prec);
-//      double nrm2 = norm_2(spinorCheck, Vh*spinorSiteSize, inv_param.cpu_prec);
-//      double src2 = norm_2(spinorIn, Vh*spinorSiteSize, inv_param.cpu_prec);
-//      double l2r = sqrt(nrm2 / src2);
-//
-//      printfQuda("Shift %d residuals: (L2 relative) tol %g, QUDA = %g, host = %g; (heavy-quark) tol %g, QUDA = %g\n",
-//		 i, inv_param.tol_offset[i], inv_param.true_res_offset[i], l2r,
-//		 inv_param.tol_hq_offset[i], inv_param.true_res_hq_offset[i]);
-//    }
-//    free(spinorTmp);
-//  }
-//    #if 0
-//   else
-  for (int i=0; i <inv_param.num_src; i++){
+  printfQuda("\nDone: %i iter / %g secs = %g Gflops, total time = %g secs\n", inv_param.iter, inv_param.secs,
+             inv_param.gflops / inv_param.secs, time0);
+
+  for (int i = 0; i < inv_param.num_src; i++) { invertQuda(spinorOutMulti[i], spinorIn[i], &inv_param); }
+
+  //  if (true) {
```

Comment:
is this dead code?

Reply:
https://github.com/lattice/quda/commit/29dc0b5b91836824b59a9b96b4555fda2be45285